Tizen Update

CazuzaCzz opened the pull request:
3.6.3
- Changed version to 3.6.3
- Changed to Required version 2.3 on config.xml
- Added some STR for languages changes. Used what twitch use on their
  site, don't know if its the better translation
- Didn't test DE and RU languages on my TV. It should work.
- Added support to follower.
- Added KEY_TOOLS to open window to input username and open follower
  window.
- Added KEY_0 to open follower window with saved username. If there is
  no username save, it will open window to input username.
- You can input username on the app, and see list of streams, live
  hosts, and live games which that that user is following.
- The username should save on WebStorage, so when you close and open the
  app, it should still be there.
- You can press the key TOOLS to open interface where you can input the
  username, and save/open.
- if there there is already a username you can pres the KEY_0 to open
  the list whithout typing again the username.

Commit:
Now displays why it failed.

Commit:
Uniform quotation marks

CazuzaCzz committed:
tizen update

tizen update

CazuzaCzz committed:
Added Key 3 and 6

Key 3 and 6 will do what the key KeyUp and KeyDown should be doing.

CazuzaCzz committed:
register key fix

add tizen.tvinputdevice.registerKey("6");
change icon

CazuzaCzz committed:
fix for arrowup and arrowdown

Added preventDefault() to stop keys from doing default actions
Removed Key 3 and 6

CazuzaCzz committed:
languague suport and some changes

- Added language support.(RU localization need test).
- Started to add support for Follower
- Added license to config.xml <license xml:lang="en-gb"
href="http://www.gnu.org/licenses/gpl.html">GNU GENERAL PUBLIC
LICENSE</license>
- Changed organization on index.html. Used Code Beautifier.

CazuzaCzz committed:
Changed version to 3.6.3

3.6.3
- Changed version to 3.6.3
- Changed to Required version 2.3 on config.xml
- Added some STR for languages changes. Used what twitch use on their
site, don't know if its the better translation
- Didn't test DE and RU languages on my TV. It should work.
- Added support to follower.
- Added KEY_TOOLS to open window to input username and open follower
window.
- Added KEY_0 to open follower window with saved username. If there is
no username save, it will open window to input username.
- You can input username on the app, and see list of streams, live
hosts, and live games which that that user is following.
- The username should save on WebStorage, so when you close and open the
app, it should still be there.
- You can press the key TOOLS to open interface where you can input the
username, and save/open.
- if there there is already a username you can pres the KEY_0 to open
the list whithout typing again the username.

PaulFurlet committed:
Create uk.js

Ukrainian language module.
There is apostrophe in STR_ERROR_NETWORK_DISCONNECT, do not know if it must be "shielded" with \. Please add it if required.

Commit:
Update es.js

CazuzaCzz committed:
Change Return key behaivor

Change Return key behaivor:
- on main window, close app
- on other windows return to main window.
Fixed refresh key on follower window

Szedar committed:
Polish language

Add a Polish language.

CazuzaCzz committed:
Changed error dialog

Changed error Dialog when Channel or User name does not exist.
Changed some behaivor from Return key when dialog error is show.

CazuzaCzz committed:
Changes to loadDataError and behaivor on connection lost

Added Network.public privilege to get network changes.
Removed Previus changes from SceneChannel loadDataError.
When connection lost, it will wait until finisht playing the downloaded
video, then it will close the AvPlayer.
Added net Languages STR variables.
Added way to check if SceneBrowser or Channel is on focus.
Changed loadDataError:
- When you try to open Channel with Open, it will check if Username or
Channel exist.
- Will tell when the stream is offline.
Added addNetworkStateChangeListener();
- It will check when the network is disconnected, and show it to de
user.

CazuzaCzz committed:
Small Changes

Changed shutdownStream() to corret listener.
Added NetworkStateChangedCallback to response when wired network change.
Removed new icon, using old one form icon folder.
Changed jquery to  new version and to min.js

CazuzaCzz committed:
Update version and README

Update version and README

CazuzaCzz committed:
Changed languages.js files

Changed languages.js files to look like master repository

CazuzaCzz committed:
Merge branch 'patch-1' of https://github.com/Jiimiex/smarttv-twitch into Jiimiex-patch-1

CazuzaCzz committed:
Merge branch 'Jiimiex-patch-1' into tizen

CazuzaCzz committed:
Merge branch 'patch-1' of https://github.com/PaulFurlet/smarttv-twitch into PaulFurlet-patch-1

CazuzaCzz committed:
Merge branch 'PaulFurlet-patch-1' into tizen

CazuzaCzz committed:
Changed uk .js file name and folder, add missing STR

CazuzaCzz committed:
Update README.md

CazuzaCzz committed:
Merge branch 'patch-1' of https://github.com/Szedar/smarttv-twitch into Szedar-patch-1

CazuzaCzz committed:
Merge branch 'Szedar-patch-1' into tizen

CazuzaCzz committed:
Changed lang\pl.js filename and folder

Fixed - Changed 3 de STR language

CazuzaCzz committed:
es language fiz

CazuzaCzz committed:
Version 3.6.5

- Changed to version 3.6.5
- Added support for Multitasking.
- If the user change to Smart Hub while playing one stream it will stop
the app, and pause the AvPlayer. When the player return to the app, it
will resume the stream.
- If the user change to Smart Hub when strem list, it will refresh the
page.
Added support to save defaultQuality.
- Default defaultQuality is "High", to change it, the user have to press
Key_9 on the desired new defaultQuality, on Stream Info Panel, where the
user can change stream quality.
- While there is no network, only the return or exit key will work.
- Changed window.load to document.addEventListener("DOMContentLoaded").
To load the app faster.
- Changed Behaivor of addNetworkStateChangeListener.

pietrogalassi committed:
Create language.js

italian localization

mkvd commented:
Hi there,

You did a great job but why do you think Tizen version should be pulled into this one?
They seem more like different projects to me.

Regards,
Nikita Katchik

midN committed:
Minor changes to make app work on 2016 Tizen OS TVs

midN committed:
Change initial quality to Source due to player not loading.

midN committed:
Remove forced 4K since some TVs might not support it.

midN committed:
Take always 0 element from qualities array. ( Highest )

pedrro committed:
Fix typo in pt-br language

whitedragonLT committed:
Update SceneBrowser.js

whitedragonLT committed:
Update SceneChannel.js

whitedragonLT committed:
Update SceneChannel.js

CazuzaCzz committed:
Revert "Take always 0 element from qualities array. ( Highest )"

This reverts commit 1c18f8b47882d72680b936bc386fbb55463bb39c.

CazuzaCzz committed:
Removed Quality Auto

Removed the auto option from quality list, that was not working. Dont
know how to use quality auto now.

If default quality, is not available, it will use the fist quality
found.
You can set default quality pressing KEY 9 when choosing quality.
